tests/nnapi/specs/V1_2/sub_dynamic_nnfw.mod.py

```diff
@@ -0,0 +1,36 @@
+# model
+
+import dynamic_tensor
+
+model = Model()
+
+model_input1_shape = [3, 4]   # first input shape of Sub. 12 float32s
+
+dynamic_layer = dynamic_tensor.DynamicInputGenerator(model, model_input1_shape)
+
+test_node_input = dynamic_layer.getTestNodeInput() # first input of Sub is dynamic tensor
+
+i2 = Input("op2", "TENSOR_FLOAT32", "{1, 4}") # second input of Sub. 4 float32s
+t1 = Internal("op3", "TENSOR_FLOAT32", "{1}") # result of first Sub1. dynamic and shape is not known
```

Comment:
Please use `{}` for a tensor of unknown rank. https://github.com/Samsung/ONE/pull/1318/files